nailgun/entities.py

```diff
@@ -899,6 +899,35 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(AbstractComputeResource, self).__init__(server_config, **kwargs)
 
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+        The format of the returned path depends on the value of ``which``:
+
+        available_images
+            /api/compute_resources/:id/available_images
+        available_flavors
+            /api/compute_resources/:id/available_flavors
+        available_zones
+            /api/compute_resources/:id/available_zones
+        available_networks
+            /api/compute_resources/:id/available_networks
+        images
+            /api/compute_resources/:id/images
+
```

Comment:
Please add support for below as well :
``
 /api/compute_resources/:id/associate
``